src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,150 @@
+import { useEffect, useMemo, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+import { PeopleTable } from '../PeopleTable';
+import { Loader } from '../Loader';
+import { PeopleFilters } from '../PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { Params } from '../../types/Params';
+import { getSearchWith } from '../../utils/getSearchWith';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = useMemo(() => {
+    return searchParams.get('sex') || '';
+  }, [searchParams]);
+
+  const query = useMemo(() => {
+    return searchParams.get('query') || '';
+  }, [searchParams]);
+
+  const centuries = useMemo(() => {
+    return searchParams.getAll('centuries') || [];
+  }, [searchParams]);
+
+  function setSearchWith(paramsToUpdate: Params) {
```

Comment:
It is important to stick to the same way of functions everywhere, fix it everywhere
```suggestion
  const setSearchWith = (paramsToUpdate: Params) => {
```